scripts/reborn_webui_v2_live_qa/run_live_qa.py

```
@@ -783,21 +783,210 @@ async def start_reborn_server(
         page = await context.new_page()
+        diagnostics.attach_page(page)
         try:
             await action(page)
         except Exception:
             screenshot = output_dir / f"{case_name}.failure.png"
             await page.screenshot(path=str(screenshot), full_page=True)
+            diagnostics.screenshot_path = screenshot
+            try:
+                await context.tracing.stop(path=str(diagnostics.trace_path))
+                diagnostics.trace_written = True
+            except Exception as trace_exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)